test-utils/src/commonMain/kotlin/mock/focus/MockFocusHandler.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.test.mock.focus
+
+import com.splendo.kaluga.keyboard.FocusHandler
+import kotlinx.coroutines.flow.MutableStateFlow
+
+open class BaseMockFocusHandler {
+    private val _isFocused = MutableStateFlow(false)
+    val isFocused: Boolean
+        get() = _isFocused.value
+
+    protected fun giveFocus() {
```

Comment:
I would choose a name that explicitly says that it is a method of the mock. mockFocused or simulateFocused for instance. Not sure what is a better name but currently it is not clear where is a method of the interface and where is mock

Reply:
[2705a80](https://github.com/splendo/kaluga/pull/467/commits/2705a80b8b2ca617d6ec22404e68338dddbc70af)
I added 2 method to the expect class and using those to give/remove the focus.